pystan/plots.py

```diff
@@ -16,4 +16,4 @@ def traceplot(fit, pars, **kwargs):
     except ImportError:
         logger.critical("matplotlib required for plotting.")
         raise
-    return plots.traceplot(fit.extract(), pars, **kwargs)
+    return plots.traceplot(fit.extract(dtypes=dtypes), pars, **kwargs)
```

Comment:
My idea was that if someone uses pystan.plots to call traceplot, then after this PR they need to include {} explicitly. If default is `None`, then no changes are needed for the current codes.